src/screens/Home/HomeScreen.tsx

```diff
@@ -14,18 +14,23 @@ export function HomeScreen({navigation}: ScreenProp<'Home'>) {
     setHasProblemGettingPermissions,
   ] = useState(false);
 
-  async function pickContact() {
+  async function startAddPlant() {
     const contact = await selectContactLib.selectContact();
     if (contact) {
       navigation.navigate('PlantChoice', {
         contact,
+        action: 'addPlant',
       });
       setHasProblemGettingPermissions(false);
     } else {
       setHasProblemGettingPermissions(true);
```

Comment:
this leads to the error message when user cancels adding plants. is this necessary? 

<img width="325" alt="Screen Shot 2021-01-03 at 7 17 43 PM" src="https://user-images.githubusercontent.com/40501912/103486794-5c422900-4df8-11eb-8121-2d2ce23ac9f4.png">

Reply:
the same behavior is on master, I agree it should be fixed.

Opened an issue: https://github.com/mmxw/Forester/issues/31